### PR TITLE
Return empty `string` when there are no arguments

### PR DESCRIPTION
**Pull request by kevva**
Fixes https://github.com/chalk/chalk/pull/182#issuecomment-317263716.

### CODE REVIEW

**Comment by Qix-** [comment]
Could just fix this by doing `if (argsLen === 0) return '';`, and skip a bunch of branching below.

File: index.js [hunk 1]
```diff
@@ -154,6 +154,10 @@ function applyStyle() {
 	const argsLen = args.length;
 	let str = argsLen !== 0 && String(arguments[0]);
```
Comment (kevva) [inline index.js]: This check is probably moot now, want me to remove it?

Comment (Qix-) [inline index.js]: Yeah should be fine. `String(arguments[0])` is going to evaluate to `String(undefined)`, which will just evaluate to `"undefined"` - that is, in the case that `argsLen === 0`.

Comment (kevva) [inline index.js]: Done.

**Pull request merged**